src/style.css

```diff
@@ -0,0 +1,103 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 900;
+}
+
+body {
+  padding-top: 12px;
+}
+
+.form-block {
+  position: relative;
+}
+
+.form-block:not(:last-child) {
+  margin-bottom: 20px;
+}
+
+.search-area {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  background: #fff;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search-area-large {
+  padding: 25px 0 26px 62px;
+  box-sizing: border-box;
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search-area-small {
+  padding: 11px 0 11px 33px;
+  box-sizing: border-box;
+  height: 42px;
+  font-size: 14px;
+  line-height: 19px;
+}
```

Comment:
You didn't fix previous comment (check point 2 in [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) )
You need to set some styles as default to search-area class, and then change needed properties to search-area-small class.
search-area-large class is redundant, you can remove it